app/models/user.rb

```diff
@@ -342,7 +342,7 @@ def fetched_avatar(size = 100)
   end
 
   def refresh_counts!
-    Keystore.put("user:#{self.id}:stories_submitted", self.stories.not_deleted.count)
```

Comment:
[I'm transcribing information that was given to me, forgive any lossiness in the process.]

As I understand it, changing this settings will effect how karma is calculated:

https://github.com/lobsters/lobsters/blob/791b21f60a2e952cabe5187559c8fb7724b7b8aa/app/models/user.rb#L202-L208

In that we will no longer account for karma on deleted stories.  That's not appropriate, deleted stories should still have their karma accounted for.  Is it possible to update the user page to use stories_displayed, rather than changing the way we count stories_submitted?

Reply:
Aren't we using `stories_displayed` already?

 https://github.com/lobsters/lobsters/blob/2d1887cb35f761b7c7fdf95adce74003e19511d0/app/views/users/show.html.erb#L90

As far as I understand, the issue here is that when `UsersHelper#stories_submitted_content` is called, we compute the stories displayed as `stories_displayed = stories_submitted - stories_deleted`, but `User#refresh_counts!` already _substracted_ the deleted stories by counting only the not deleted ones.

https://github.com/lobsters/lobsters/blob/2d1887cb35f761b7c7fdf95adce74003e19511d0/app/models/user.rb#L342

Another option is to subtract `User#stories_deleted_count` when the karma is calculated. 

```ruby
k.to_f / (self.stories_submitted_count - self.stories_deleted_count + self.comments_posted_count)
```

This approach would make the way the karma is calculated clearer.

Reply:
By the way, thanks for the quick feedback!